### DISCUSSION TITLE
unsupported on-disk structure for file - is there a table with compatible versions?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [mariuz/flamerobin#361] [General]
unsupported on-disk structure for file - is there a table with compatible versions?

**Comment by arvanus** [comment]
Here you go:

https://ib-aid.com/en/articles/all-firebird-and-interbase-on-disk-structure-ods-versions/

Server version|ODS|Can work with ODS|Details
-- | -- | -- | -- 
InterBase6.0<br>Firebird 1.0<br>Yaffil 1.0 | 10.0 | 9.0/9.1 | It is dangerous to work with ODS 9.x, because new InterBase and  Firebird use new metadata format, which is not recognized by InterBase  5.x
Firebird 1.5 | 10.1 | 9.0/9.1/10.0 | ODS 10.1 of 64bit Firebird 1.5 is incompatible to 32bit ODS 10.1.  This is the only one known incompatibility of database format between  32/64bit versions.
Firebird 2.0 | 11.0 | 10.x | ODS 11 of Firebird 2.0 is incompatible to InterBase 7.x
Firebird 2.1 | 11.1 | 10.x | ODS 11 of Firebird 2.0 is incompatible to InterBase 7.x
Firebird 2.1 | 11.1 | 10.x/11.0 |  
Firebird 2.5 | 11.2 | 10.x/11.x | ODS 11.2 is incompatible with Firebird 2.0/2.1.
Firebird 3.0 | 12.0 | Does not support previous ODS |  
Firebird 4.0 | 13.0 | Does not support previous ODS |  
Firebird 5.0 | 13.1 | 13.0, 13.1 | Database can be upgraded from 13.0 to 13.1 with gfix -upgrade from 13.0, or with backup/restore

**Comment by Jdochoa** [comment]
> unsupported on-disk structure for file - is there a table with compatible versions?

You problem is the version of fbclient.

./jo
